src/main/java/net/coderbot/iris/pipeline/DeferredWorldRenderingPipeline.java

```diff
@@ -199,6 +200,11 @@ public DeferredWorldRenderingPipeline(ProgramSet programs) {
 			return shadowMapRenderer;
 		};
 
+		this.prepareRenderer = new CompositeRenderer(programs.getPackDirectives(), programs.getPrepare(), renderTargets,
```

Comment:
`gbuffers_terrain` might be reading from the wrong flipped buffers with this current behavior. We'll need to split `flippedBeforeTranslucent` into `flippedBeforeShadow` and `flippedAfterPrepare`, and then update references accordingly. Also, `flippedBeforeTerrain` references need to be updated to either `flippedBeforeShadow` or `flippedAfterPrepare` as appropriate.

Reply:
Will try to implement this today

Reply:
Fixed.